x/tokenfactory/keeper/genesis_test.go

```diff
@@ -1,14 +1,35 @@
 package keeper_test
 
 import (
+	"testing"
+
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	"github.com/stretchr/testify/suite"
 	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
 
+	"github.com/osmosis-labs/osmosis/v11/app/apptesting"
 	"github.com/osmosis-labs/osmosis/v11/x/tokenfactory/types"
 )
 
-func (suite *KeeperTestSuite) TestGenesis() {
+type GenesisTestSuite struct {
+	apptesting.KeeperTestHelper
+}
+
+func (suite *GenesisTestSuite) SetupTest() {
+	suite.Setup()
+
+	// remove module account to ensure initGenesis initializes it on its own
+	moduleAddress := suite.App.AccountKeeper.GetModuleAddress(types.ModuleName)
+	tokenfactoryModuleAccount := suite.App.AccountKeeper.GetAccount(suite.Ctx, moduleAddress)
+	suite.App.AccountKeeper.RemoveAccount(suite.Ctx, tokenfactoryModuleAccount)
+}
+
+func TestGenesisTestSuite(t *testing.T) {
+	suite.Run(t, new(GenesisTestSuite))
+}
+
+func (suite *GenesisTestSuite) TestGenesis() {
```

Comment:
I think it would be better to just define this as a part of the`KeeperTestSuite`. No need to make a whole new test suite for a single action

Reply:
Fixed!